postcss.mdcss.js

```diff
@@ -23,7 +23,7 @@ module.exports = require('mdcss')({
       'octicons/octicons.css',
       'https://fonts.googleapis.com/css?family=Roboto:400,500,700',
       `public/doc.css`,
-      `base-styles/base-styles.full.css`,
+      `base-styles.full.css`,
```

Comment:
this fixes a weird issue with `fds-styles` docs including the custom normalize CSS twice, which made things look weird in firefox/safari